lib/sagas/requests/index.js

```diff
@@ -250,13 +260,22 @@ export function * messageHandler ({message, popup, target, redirectUrl, postback
     const requestType = JWT_REQUEST_TYPE[payload.type] || 'attestation'
     const handler = HANDLERS[requestType]
     const boxPub = payload.boxPub // Breaking change, may introduce later: ->  || (doc.publicKey.find(pk => pk.type === 'Curve25519EncryptionPublicKey') || {}).publicKeyBase64
+    
     if (handler && handler.handle) {
       const request = yield call(handler.handle, payload, message, id)
       if (request) {
         request.type = requestType
         if (redirectUrl) request.redirect_url = redirectUrl
+        if (callbackUrl) request.callback_url = callbackUrl
         if (postback !== undefined) request.postback = postback
         let callback = request.callback_url ? URL(request.callback_url) : undefined
+
+        if (requestType === 'attestation') {
+          if (request.callback_url && postback) {
+            yield spawn(performCallback, request, { status: 'ok' })
```

Comment:
@rmw2 Not sure should I send a response body here? Is Chasqui expecting a body?

Reply:
It's posting back but the QR code is not being removed

Reply:
Unclear exactly what chasqui is expecting because this whole flow is underspecified.  Though it would be a hack, the key `access_token` is already recognized, and will allow a chasqui polling transport to resolve, so maybe we could just toss a `{access_token: 'accepted'}` or something like that as the body ?

Again this is all stuff we need to fix with a more thorough messaging spec, so this is just trying to do the least effort/least damage patch to get something working

Reply:
Tried that and get this error from Chasqui. Seems its looking for an ID.
<img width="537" alt="screenshot 2018-12-06 at 11 41 13" src="https://user-images.githubusercontent.com/7109973/49582114-e1be2b00-f94b-11e8-9dd7-86800c7f7891.png">

Reply:
Hm, this would suggest that the chasqui topic wasn't created before posting to it -- is this after receiving a request from a QR on the demo app?

Reply:
Yeah this happens after scanning the QR code. It's calling back but it's expecting something more specific in the body.

Reply:
Knowing the chasqui code, I don't think it's the body -- The 404 is returned when you try to post to a non-existant topicId.  I think there was also some shenanigans around deleting topics after 60s, is there any chance it's longer than that between opening the QR and trying to post from mobile ?

Reply:
Hmm also this is using the old chasqui URL in the QR -- that's probably the fault of `connect`/the demo app.  Maybe something different happens in the old chasqui code, but [this](https://github.com/uport-project/lambda-chasqui/blob/develop/src/api/topic_post.js#L25) is the equivalent condition in `lambda-chasqui`

Reply:
Nope, literally scanning straight away. Is it possible Chasqui just doesn't accept callbacks other than the login callback?

Reply:
Definitely shouldn't be the case --  chasqui is made to CRUD arbitrary data to a "topic", which gets a unique URL.  POSTing to the topic should update the data with whatever is in the body of the post, and GETing it should give back the data stored at the topic.  I'll read through the old chasqui code to check if there is any access control or something else weird that could be happening, but this is confusing

Reply:
Ok [here](https://github.com/uport-project/chasqui/blob/master/src/api/topic.js#L57) is the error, still unclear why it would be happening

Reply:
Ah ok one potential issue: chasqui topic ids are supposed to be 16 urlbase64 characters, and this url is 22.  Maybe something is going wrong with the parsing of the callback url param?

Reply:
Looks like 22 characters is what's coming in?

<img width="792" alt="screenshot 2018-12-11 at 10 47 31" src="https://user-images.githubusercontent.com/7109973/49795621-5faa7980-fd32-11e8-925e-2f3d28982d63.png">